### ISSUE TITLE
Put CJ on the staff page!

### ISSUE THREAD

**Issue by mekline** [lookit/lookit-api#857]
**TL;DR**: Update 'meet the scientists' page with current staff!

Please replace Rico with CJ and Kim with Melissa. Add a new section for 'staff alumni', above the 'student alumni' section, to put Kim & Rico in. 

** Details**
- Use the photos of CJ & Melissa that are at https://eccl.mit.edu/meet-the-lab

Change 1: 
Rico picture --> CJ picture
Rico Rodriguez --> CJ Green
Rico's blurb --> "CJ is Lookit's lead software infrastructure engineer, working on planning and adding features for both participants and researchers" (Feel free to modify if you want!)

Change 2:
Kim picture --> Melissa picture
Kim Scott --> Melissa Kline Struhl
Kim's blurb --> "Melissa Kline Struhl is the director of Lookit and a research scientist in the MIT Early Childhood Cognition Lab. She loves it when the data surprises her, and loves talking to researchers and families about how to make Lookit better."

Change 3:
New section, above and formatted identically to the "Undergraduate student alumni" section.

Title: Staff alumni

Once circle each for Kim & Rico, reusing their pictures from above. Their names should be large (same as undergrad section).  Underneath, put titles: 

Kim Scott
Founder and Director

Rico Rodriguez
Software Engineer

**Comment by okaycj** [comment]
This is a flatpage, so I updated the page in staging.  Here's the link https://lookit-staging.mit.edu/scientists/.

**Comment by okaycj** [comment]
These changes are now in production.  Closing issue.
